src/components/Navbar.tsx

```diff
@@ -1,43 +1,158 @@
+import Settings from '@mui/icons-material/Settings';
+import Logout from '@mui/icons-material/Logout';
+import { getCurrentUser } from "@/utils";
+import { User } from "@/model";
+import { freelancerExists } from "@/redux/services/freelancerService";
+import { getFreelancerProfile } from "@/redux/services/freelancerService";
+import defaultProfile from "../assets/images/profile-image.png"
+import MenuItems from "./Navbar/MenuItems";
+import Login from "./Login";
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+import { useRouter } from "next/router";
 
 function Navbar() {
-  const [sideBarIsVisivle, setSideBarIsVisible] = useState<boolean>(false);
+  const [loginModal, setLoginModal] = useState<boolean>(false);
+  const [redirectURL, setRedirectURL] = useState<string>();
+  const [freelancerProfile, setFreelancerProfile] = useState<any>()
+  const [user, setUser] = useState<User>()
+
+  const [solidNav, setSolidNav] = useState<boolean>(false)
 
-  const toggleSideBar = (): void => {
-    setSideBarIsVisible(!sideBarIsVisivle);
+  const router = useRouter()
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const [openMenu, setOpenMenu] = useState(false)
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(Boolean(event.currentTarget))
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+    setOpenMenu(false)
   };
+
+  useEffect(() => {
+    const setup = async () => {
+      const user = await getCurrentUser() || false;
+      if (user) {
+        const res = await getFreelancerProfile(user.username)
+        setFreelancerProfile(res);
+        setUser(user);
+      }
+    };
+    setup();
+  }, []);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const isScrolled = window.scrollY > 50
+
+      if (isScrolled) setSolidNav(true)
+      else setSolidNav(false)
+    }
+
+    document.addEventListener('scroll', handleScroll)
+    return () => {
+      document.removeEventListener('scroll', handleScroll)
+    }
+  }, [])
+
+  const navigaeToPage = () =>{
+    if(user?.username){
+      router.push("/briefs/new")
+    }
+    else{
+      setLoginModal(true)
+    }
+  }
+
+
   return (
     <>
-      <header className="py-[15px] lg:py-[30px]" id="header-wrapper">
-        <div id="main-header">
+      <header className={`navBar ${solidNav ? "bg-theme-black-text" : "bg-transparent"}`} id="header-wrapper">
+        <div className="text-center w-full bg-primary text-black py-1 text-xs lg:text-sm">
+          Thanks for trying the beta version of Imbue. Please let us know what we should work on to make it better! Submit your feedback
+          <a href="https://pfljr3ser45.typeform.com/to/bv00pviH" target="_blank" className="underline cursor-pointer ml-1">here</a>
```

Comment:
`https://pfljr3ser45.typeform.com/to/bv00pviH`

this link should be configurable.